src/apps/vguest/vguest_app.lua

```diff
@@ -0,0 +1,77 @@
+
+module(..., package.seeall)
+local pci       = require("lib.hardware.pci")
```

Comment:
`pci` is not used. Nit: leave a space between the imported headers and the module definition.